textRevers/src/main/java/app/textRevers/Main.java

```diff
@@ -101,4 +104,19 @@ private static void askToContinue(){
 		}
 	}
 
+	/**
+	 * Method name: charsCounterExecutor
+	 * <p>
+	 * <p>
+	 * Inside the function: Print simple interface to insert the world/sentence to count chars.
+	 */
+	private static void charsCounterExecutor(){
+		System.out.println( "* * * * * * * * * * * * * * * * * *\n"+
```

Comment:
For the repeating strings you should create local variables in class and use them. When you use new Strings written in println java creates new object of String class. Imagine how much memory it will use in continuous usage